src/includes/enriching-events/attach-screenshots/unity.mdx

```diff
@@ -0,0 +1,13 @@
+The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
+
+</Note>
+
+### Max Width and Max Height
```

Comment:
Please amend the specs with this 2 configs https://develop.sentry.dev/sdk/features/#screenshots

Reply:
I'd also add this new 2 options in here below https://docs.sentry.io/platforms/android/configuration/options/#attach-screenshot